lib/cuke_linter/linters/background_does_more_than_setup_linter.rb

```diff
@@ -4,17 +4,33 @@ module CukeLinter
 
   class BackgroundDoesMoreThanSetupLinter < Linter
 
+    # Changes the linting settings on the linter using the provided configuration
+    def configure(options)
+      @when_keywords = DialectHelper.get_configured_keywords(options, DEFAULT_WHEN_KEYWORD)
+      @then_keywords = DialectHelper.get_configured_keywords(options, DEFAULT_THEN_KEYWORD)
+    end
+
     # The rule used to determine if a model has a problem
     def rule(model)
       return false unless model.is_a?(CukeModeler::Background)
 
-      model.steps.collect(&:keyword).any? { |keyword| (keyword == 'When') || (keyword == 'Then') }
+      model.steps.collect(&:keyword).any? { |keyword| when_keywords.include?(keyword) || then_keywords.include?(keyword) }
```

Comment:
...these just pick up the `@`...
```
@when_keywords.include?(keyword) || @then_keywords.include?(keyword)
```

Reply:
humm... using in this way the @when_keywords is nil 😕 
I'll find out what happened

Reply:
Oh, right. Because when `configure` is never called then a default isn't set. I'd have run into that in a few minutes when I started to refactor after adding more unit tests. Maybe we'll still need those private wrapper methods for the keywords after all.